### PR TITLE
Added number of comments per issues on issue page

### PR DESCRIPTION
**Pull request by mohitanand001**
Now the issues page would include the number of comments on each issues ,and clicking on each link would lead to the respective issue.
screenshot from 2017-06-15 21 57 44

### CODE REVIEW

File: website/static/css/style.css [hunk 1]
```diff
@@ -452,4 +452,9 @@ input[type='file'] {
 .bug-type {
 	position: absolute;
 	right: 20px;
+}
+.comment_link {
+ 	position:absolute;
+ 	bottom:10px;
+ 	left:500px;
 }
```
Comment (souravbadami) [inline website/static/css/style.css]: Add a new line here at the end.

File: website/templates/domain.html [hunk 2]
```diff
@@ -123,6 +123,11 @@
                                     <span class="text-muted small">
                                         <em>{{ activity.created|timesince }} ago</em>
                                     </span>
+
+                                     <a  class="comment_link" href="{{activity.get_absolute_url }}#comments_form">
```
Comment (souravbadami) [inline website/templates/domain.html]: Remove the single space after **<a**.

**Review by souravbadami — CHANGES_REQUESTED** [review]
Hi @farziengineer -- I've done a first pass. I think, we can extend this to the homepage as well. What do you think ?

**Comment by farziengineer** [comment]
Sure why not. I'll do a fix

File: website/templates/_activity.html [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {% load gravatar %}
 <div class="list-group-item activity-strip">
     <div class="activity-strip-section">
+    <a  class="comment_link" href="{{activity.target.get_absolute_url }}#comments_form">{{ activity.target.comments.all|length }} comments</a>
```
Comment (souravbadami) [inline website/templates/_activity.html]: Remove the single space after <a.

**Comment by souravbadami** [comment]
@farziengineer -- This looks good. I have added some additional comments on the thread. We can merge this once you address them.

File: website/static/css/style.css (hunk 1 above)
Comment (farziengineer) [inline website/static/css/style.css]: After the brace "}" ?

**Review by souravbadami — APPROVED** [review]

**Pull request merged**